app/views/funding/payment_schedules/show.html.erb

```diff
@@ -1,6 +1,5 @@
 <%= render PageTitle::View.new(text: "Funding") %>
```

Comment:
Can we update the page title to "Funding - Payment schedule 2021 to 2022", where the 2021 - 2022 is dynamic?

Reply:
We probably also need to add this to the locale file to be consistent with other pages (Steve did some work recently to clean this up)

Reply:
i updated this in my other bug PR when i made the trainee summary title dynamic so i didn't clash with translations: https://github.com/DFE-Digital/register-trainee-teachers/pull/2392/files#diff-a083ed40b68a0be53600e0cd6a6afdf920e566d157c93b88b1d814e4b8aa7ddd

Reply:
nice one :)